Dockerfile.arm

```diff
@@ -27,10 +27,29 @@ RUN dotnet publish Jellyfin.Server --configuration Release --output="/jellyfin"
 
 FROM multiarch/qemu-user-static:x86_64-arm as qemu
 FROM arm32v7/debian:buster-slim
+
+ARG DEBIAN_FRONTEND="noninteractive"
```

Comment:
For `ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn` it's because of this for the curling keys.
Warning: apt-key output should not be parsed (stdout is not a terminal)
https://stackoverflow.com/questions/48162574/how-to-circumvent-apt-key-output-should-not-be-parsed 

The other one, I forget why

Reply:
I removed noninteractive without any issues so i gutted it.

Reply:
`apt` being interactive would cause issues when we install a package that has some initial setup if installed in interactive mode (like `mariadb` for instance). I still say keep it.

Reply:
ok, adding `ARG DEBIAN_FRONTEND="noninteractive"` back in

Reply:
These should have comments to ensure everyone knows why.